tests/main_test.cpp

```diff
@@ -1,275 +0,0 @@
-#include "vectorcxx/src/lib.rs.h"
```

Comment:
I delete the entire test since it is covered by the topology_test now. We should make all the tests repeatable